src/index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable*/
```

Comment:
- **_Fix the lint error(or disable an error, avoid overuse of linter disable) instead of disabling eslint on the whole file_**